client/src/components/ReviewTile.js

```diff
@@ -12,11 +14,77 @@ const ReviewTile = (props) => {
     }
   })
 
+  const handleClick = async (event) => {
+    if(props.userId) {
+      setShowVoteError(false)
+      let alreadyVoted = false
+      props.votes.forEach(vote => {
+        if(vote.userId === props.userId) {
+          alreadyVoted = true
+          setShowVoteError(true)
+        }
+      })
+      if(!alreadyVoted) {
+        try {
+          const response = await fetch(`/api/v1/reviews/${props.reviewId}/vote`, {
+            method: "POST",
+            headers: new Headers({
+              "Content-Type": "application/json"
+            }),
+            body: JSON.stringify({ userId: props.userId, direction: event.currentTarget.value })
+          })
+          if(!response.ok){
+            const errorMessage = `${response.status}: (${response.statusText})`
+            const error = new Error(errorMessage)
+            throw(error)
+          } else {
+            const body = await response.json()
+            props.addNewVote(body.newVote, props.reviewId)
+          }
+        } catch(err) {
+          console.error(`Error in fetch: ${err.message}`)
+        }
```

Comment:
for the sake of DRYness, I would add this POST fetch to its own function defined in another place, rather than adding it to the already busy `handleClick` function